test/runtests.jl

```diff
@@ -85,10 +81,11 @@ cd(@__DIR__) do
 
     if use_revise
         @everywhere begin
+            @eval using Revise
             Revise.track(Core.Compiler)
             Revise.track(Base)
             for (id, mod) in Base.loaded_modules
-                if id.name in STDLIBS
+                if id.name in filter!(x -> isfile(joinpath(Sys.STDLIB, x, "src", "$(x).jl")), readdir(Sys.STDLIB))
```

Comment:
Do the readdir only once? As written it's done `nworkers * length(Base.loaded_modules)` times.

Reply:
Actually, I think it would be better to just serialize the STDLIBS we already have here rather than re-compute it. Doesn't distributed allow something like `$STDLIBS` to refer to the local one?

Reply:
Should be better now. Looks like we weren't using Revise on restarted workers, so this fixes that too.